lib/ProductOpener/Products.pm

```diff
@@ -2946,6 +2947,21 @@ or "slack_CHANNEL_NAME" (B<warning> currently channel name is ignored, we post t
 
 =cut
 
+sub preprocess_product_field ($product_ref) {
+
+	my @tag_fields = qw(brands categories origins labels packaging stores manufacturing_places);
+	foreach my $field (@tag_fields) {
+		if (defined $product_ref->{$field}) {
+			my @tags = split(/,/, $product_ref->{$field});
```

Comment:
This works, but you could call directly remove_email on $product_ref->{$field} and it should give exactly the same result, without having to do a split, a foreach and a join.

Reply:
Fixing it!